wolfcrypt/src/ext_kyber.c

```diff
@@ -314,6 +313,8 @@ int wc_KyberKey_MakeKey(KyberKey* key, WC_RNG* rng)
     }
 
 #ifdef HAVE_LIBOQS
+    const char* algName = NULL;
+    OQS_KEM *kem = NULL;
```

Comment:
You can't move these here. Some compilers will have issue. Please move to top and wrap with `#ifdef HAVE_LIBOQS` or put in brace section.